src/main/webapp/js/finscholar/collegepageview/collegepageview.js

```diff
@@ -16,15 +16,19 @@
 
 goog.module('finscholar.collegepageview');
 
-const GoogDom = goog.require('goog.dom');
+const {SinglePageView} = goog.require('finscholar.singlepageview');
 const {loadCollegeData} = goog.require('datahandlers.collegepage');
 
 /** Class for the college page view. */
-class CollegePageView {
-  constructor() {}
+class CollegePageView extends SinglePageView {
```

Comment:
This class is just an empty template which renders hard coded data from server. The actual implementation is in #74 #73